docker-images/stunnel-zookeeper/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM centos:7
```

Comment:
My feeling is that once we develop the stunnel image for the kafka pods we will end up reworking this because there's not so much in this image which is specific to zk. The ports, and port mappings. So we should think about having a base-stunnel Dockerfile and deriving the ZK, Kafka and TO ones from that. But for now this is fine.

Reply:
We started with this idea then we came to the conclusion that all the first part is the same for all stunnel images and just the scripts will change. It was just a FROM, RUN and CMD which maybe didn't worth to have a base image for that. Now there are more few things but exposing the ports in the Dockerfile seems to be something useless to me. We could do a base image but let's leave that for the next PR when a stunnel will be needed for another pod.